### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by AleksandrPok**

### CODE REVIEW

File: src/main/java/injectorpractice/dao/ManufacturerDaoJdbcImpl.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create manufacturer to database "
+                    + manufacturer, e);
+        }
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String select = "SELECT * FROM manufacturer WHERE manufacturer_id = ? AND "
+                + "manufacturer_deleted = false;";
```
Comment (Resci) [inline src/main/java/injectorpractice/dao/ManufacturerDaoJdbcImpl.java]:
```suggestion
        String select = "SELECT * FROM manufacturer WHERE manufacturer_id = ? AND manufacturer_deleted = false;";
```

File: src/main/java/injectorpractice/dao/ManufacturerDaoJdbcImpl.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        Manufacturer manufacturer = null;
```
Comment (Resci) [inline src/main/java/injectorpractice/dao/ManufacturerDaoJdbcImpl.java:42]: The same
```suggestion
        Manufacturer manufacturer;
```

**Review by Resci — APPROVED** [review]

File: src/main/java/injectorpractice/dao/ManufacturerDaoJdbcImpl.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String create = "INSERT INTO manufacturer (manufacturer_name, manufacturer_country)"
```
Comment (boroda4436) [inline src/main/java/injectorpractice/dao/ManufacturerDaoJdbcImpl.java]:
```suggestion
        String create = "INSERT INTO manufacturer (name, country)"
```
You already know it is a manufacturer

File: src/main/java/injectorpractice/dao/ManufacturerDaoJdbcImpl.java [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+        String selectAll = "SELECT * FROM manufacturer WHERE manufacturer_deleted = false;";
+        List<Manufacturer> manufacturers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(selectAll)) {
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                manufacturers.add(getManufacturer(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufactorers from database", e);
```
Comment (boroda4436) [inline src/main/java/injectorpractice/dao/ManufacturerDaoJdbcImpl.java]:
```suggestion
            throw new DataProcessingException("Can't get manufacturers from database", e);
```

File: src/main/java/injectorpractice/dao/ManufacturerDaoJdbcImpl.java [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+            statement.executeUpdate();
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update manufacturer: " + manufacturer, e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String delete = "UPDATE manufacturer SET manufacturer_deleted = true WHERE "
```
Comment (boroda4436) [inline src/main/java/injectorpractice/dao/ManufacturerDaoJdbcImpl.java]:
```suggestion
        String delete = "UPDATE manufacturer SET deleted = true WHERE "
```
please check your columns naming

**Review by boroda4436 — CHANGES_REQUESTED** [review]

**Pull request merged**